package.json

```diff
@@ -110,6 +111,7 @@
     "redux-thunk": "^2.3.0",
     "reselect": "^4.0.0",
     "telepath-unpack": "^0.0.3",
+    "tippy.js": "^6.3.7",
```

Comment:
Do we actually need this in in addition to `@tippyjs/react`?

Reply:
In the future (header updates) we will need this but at this time we might not :)